### DISCUSSION TITLE
[Bug] many dust attack

### DISCUSSION THREAD

**Discussion by luis360** [Chia-Network/chia-blockchain#10595] [Support]
### What happened?

🍃 Miscellaneous | 2 minutes ago | 0xffee1ffa2a89942241701705ae2742ca1b0d41f14a4a885c23ed99781cf2be2e | 0.000000000001 XCH ($0)
-- | -- | -- | --

🍃 Miscellaneous | 2 minutes ago | 0xff5f28a2ef922aba29f6da12c624493767674202487f8631fddf326882394d38 | 0.000000000001 XCH ($0)

🍃 Miscellaneous | 2 minutes ago | 0xff4a2966ecc751586e7671f359459be0fdc66d8c6233aae9b6cca98edd3b8c7f | 0.000000000001 XCH ($0)

🍃 Miscellaneous | 2 minutes ago | 0xff3bdb0a280c4015b73c64dbff9e03d4207d55df83e91aecb6063a7c29562f5a | 0.000000000001 XCH ($0)

🍃 Miscellaneous | 2 minutes ago | 0xff328d16bc62d2b8a6fbc786d2652cb709fa4980f0b9c9e5d0e236030af6ee0f | 0.000000000001 XCH ($0)

🍃 Miscellaneous | 2 minutes ago | 0xff25b41c0efc6480d43e51eed492c1fc475d1278d65bb683f2cf6dae632ed964 | 0.000000000001 XCH ($0)

🍃 Miscellaneous | 2 minutes ago | 0xff10510f425f1f3c9b30b380d5a40dc864603b95c84880d7a62527abb66c562f | 0.000000000001 XCH ($0)

🍃 Miscellaneous | 2 minutes ago | 0xfee3e142403d0574c0be405f93b9ce6ef7156ce1048386941a6a4b1960259ef6 | 0.000000000001 XCH ($0)

🍃 Miscellaneous	[2 minutes ago](https://www.chiaexplorer.com/blockchain/coin/0xffee1ffa2a89942241701705ae2742ca1b0d41f14a4a885c23ed99781cf2be2e)	[0xffee1ffa2a89942241701705ae2742ca1b0d41f14a4a885c23ed99781cf2be2e](https://www.chiaexplorer.com/blockchain/coin/0xffee1ffa2a89942241701705ae2742ca1b0d41f14a4a885c23ed99781cf2be2e)	0.000000000001 XCH ($0)
🍃 Miscellaneous	[2 minutes ago](https://www.chiaexplorer.com/blockchain/coin/0xff5f28a2ef922aba29f6da12c624493767674202487f8631fddf326882394d38)	[0xff5f28a2ef922aba29f6da12c624493767674202487f8631fddf326882394d38](https://www.chiaexplorer.com/blockchain/coin/0xff5f28a2ef922aba29f6da12c624493767674202487f8631fddf326882394d38)	0.000000000001 XCH ($0)
🍃 Miscellaneous	[2 minutes ago](https://www.chiaexplorer.com/blockchain/coin/0xff4a2966ecc751586e7671f359459be0fdc66d8c6233aae9b6cca98edd3b8c7f)	[0xff4a2966ecc751586e7671f359459be0fdc66d8c6233aae9b6cca98edd3b8c7f](https://www.chiaexplorer.com/blockchain/coin/0xff4a2966ecc751586e7671f359459be0fdc66d8c6233aae9b6cca98edd3b8c7f)	0.000000000001 XCH ($0)
🍃 Miscellaneous	[2 minutes ago](https://www.chiaexplorer.com/blockchain/coin/0xff3bdb0a280c4015b73c64dbff9e03d4207d55df83e91aecb6063a7c29562f5a)	[0xff3bdb0a280c4015b73c64dbff9e03d4207d55df83e91aecb6063a7c29562f5a](https://www.chiaexplorer.com/blockchain/coin/0xff3bdb0a280c4015b73c64dbff9e03d4207d55df83e91aecb6063a7c29562f5a)	0.000000000001 XCH ($0)
🍃 Miscellaneous	[2 minutes ago](https://www.chiaexplorer.com/blockchain/coin/0xff328d16bc62d2b8a6fbc786d2652cb709fa4980f0b9c9e5d0e236030af6ee0f)	[0xff328d16bc62d2b8a6fbc786d2652cb709fa4980f0b9c9e5d0e236030af6ee0f](https://www.chiaexplorer.com/blockchain/coin/0xff328d16bc62d2b8a6fbc786d2652cb709fa4980f0b9c9e5d0e236030af6ee0f)	0.000000000001 XCH ($0)
🍃 Miscellaneous	[2 minutes ago](https://www.chiaexplorer.com/blockchain/coin/0xff25b41c0efc6480d43e51eed492c1fc475d1278d65bb683f2cf6dae632ed964)	[0xff25b41c0efc6480d43e51eed492c1fc475d1278d65bb683f2cf6dae632ed964](https://www.chiaexplorer.com/blockchain/coin/0xff25b41c0efc6480d43e51eed492c1fc475d1278d65bb683f2cf6dae632ed964)	0.000000000001 XCH ($0)
🍃 Miscellaneous	[2 minutes ago](https://www.chiaexplorer.com/blockchain/coin/0xff10510f425f1f3c9b30b380d5a40dc864603b95c84880d7a62527abb66c562f)	[0xff10510f425f1f3c9b30b380d5a40dc864603b95c84880d7a62527abb66c562f](https://www.chiaexplorer.com/blockchain/coin/0xff10510f425f1f3c9b30b380d5a40dc864603b95c84880d7a62527abb66c562f)	0.000000000001 XCH ($0)
🍃 Miscellaneous	[2 minutes ago](https://www.chiaexplorer.com/blockchain/coin/0xfee3e142403d0574c0be405f93b9ce6ef7156ce1048386941a6a4b1960259ef6)	[0xfee3e142403d0574c0be405f93b9ce6ef7156ce1048386941a6a4b1960259ef6](https://www.chiaexplorer.com/blockchain/coin/0xfee3e142403d0574c0be405f93b9ce6ef7156ce1048386941a6a4b1960259ef6)	0.000000000001 XCH ($0)

Very many dust attacks, my normal transactions are difficult to chain，

### Version

1.2.11

### What platform are you using?

Linux

### What ui mode are you using?

CLI

**Comment by DrakeFS** [comment]
Add the minimum fee of .00005 XCH and your transaction will go through, even during a dust storm.

**Comment by luis360** [comment]
good, thansk, but an error occurred
INVALID_FEE_TOO_CLOSE_TO_ZERO

**Comment by esaung** [comment]
what are you trying to do when you add your fee to get that error?

**Comment by luis360** [comment]
I add fee of .00005 XCH, but occasionally this error occurs when transferring coin

**Comment by emlowe** [comment]
At certain levels, depending on the type of transaction, all fees are considered as 0. So sometimes you will need to provide more fees.

**Comment by Zero545** [comment]
It seems I have the same issue currently. I can enter a fee of e.g. up to 0.005, but will still get this error message and my transaction does not get accepted (pending for more than 24h).
How do I know, what fee is actually needed to perform a successful transaction?

**Comment by emlowe** [comment]
When the mempool is full, you will need to add in a larger fee in order to remove a tx from the mempool in order to include your tx. Our fee estimator is still in progress, however, I would have expected a fee of 0.005 to work in most cases. It is completely dependent on the type of transaction and the number of coins - so there is no "one size fits all" fee.

The other issue is related to #10873, once you get this error the tx will stay pending. (although it is believed this is limited to 1.3.x)
